flink-test-utils-parent/flink-connector-test-utils/src/main/java/org/apache/flink/connector/testframe/testsuites/SourceTestSuiteBase.java

```diff
@@ -86,6 +93,8 @@
 public abstract class SourceTestSuiteBase<T> {
 
     private static final Logger LOG = LoggerFactory.getLogger(SourceTestSuiteBase.class);
+    static ExecutorService executorService =
```

Comment:
Can we wrap this `ExecutorService` into util classes?

Reply:
done